src/pages/MainPage/MainPage.jsx

```diff
@@ -1,85 +1,106 @@
+  
+  const addToCart = (product) => {
+    const curIt = {
+      product,
+      data: Date.now(),
     };
+    dispatch({ type: "ADD_TO_BASKET", payload: curIt });
+  };
 
-    function showBasket(cond) {
-        setShow(cond)
-    }
+  function showBasket(cond) {
+    window.localStorage.setItem("MY_APP_STATE", JSON.stringify(show));
+    setShow(cond);
+  }
+  useEffect(() => {
+    const data = window.localStorage.getItem("MY_APP_STATE");
+    console.log("data", data);
 
-    return (
-        <div>
-            <Container>
-                <div className={style.box}>
-                    <header className={style.box__header}>
-                        <div className={style.box__ofheader}>
-                            <div className="">
-                                {/* <img src={logo} alt="logo" /> */}
-                            </div>
-                            <div className={style.box__nameofCompany}>
-                                Bags
-                            </div>
-                        </div>
-                    </header>
-                    <main>
-                        <div className={style.box__list}>
-                            {products.map((it, i) => {
-                                return <div key={i} className={style.box__card}>
-                                    <div className="">
-                                        <i className="fa fa-heart" style={{ fontSize: 24, textAlign: 'right', color: '#fff' }} />
-                                    </div>
-                                    <div className={style.box__img}>
-                                        <img src={it.img} alt="img_bags" />
-                                    </div>
-                                    <div className="">
-                                        {it.name}
-                                    </div>
-                                    <div className="">
-                                        {it.price}
-                                    </div>
-                                    <Button className={style.button41} onClick={() => addToCart(it)}>
-                                        SHOP NOW!
-                                    </Button>
-                                </div>
-                            })}
-                        </div>
-                    </main>
-                    <div className={style.box__navigation}>
-                        <div className={style.box__search}>
-                            Шукати
-                        </div>
-                        <div className={style.box__search}>
-                            Подобається
-                        </div>
-                        <div onClick={() => showBasket(true)} className={style.box__search}>
-                            Корзина {curProducts.length}
-                        </div>
+    if (data !== null) setShow(JSON.parse(data));
+  }, []);
+  function clearHistory() {
+    window.localStorage.clear();
```

Comment:
DON"T DO THAT

Reply:
thats mean about ' ClearHistory!?'

Reply:
`clear` remove all data from storage. If u need to remove only one, please use `removeItem`. In LS u can have a lot of other informations, and after using `clear` u will lose it